proto/swagger/cluster.swagger.json

```diff
@@ -515,6 +515,13 @@
         "autoscalerOptions": {
           "$ref": "#/definitions/protoAutoscalerOptions",
           "description": "AutoscalerOptions specifies optional configuration for the Ray autoscaler."
+        },
+        "headServiceAnnotations": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
```

Comment:
I guess it's auto-generated, but do you happen to know what does `additionalProperties` mean here?
Thank you!

Reply:
Seems like it's used to represent the type of values: https://stackoverflow.com/questions/41239913/why-additionalproperties-is-the-way-to-represent-dictionary-map-in-swagger-ope
and the key type is required to be string